js/src/vercel.ts

```diff
@@ -797,83 +836,74 @@ export class AISDKExporter {
         if (seen.has(task.item.id)) continue;
 
         if (!task.item.sent) {
-          let override: OverrideRunCreate = {
-            id: task.item.id,
-            parent_run_id: task.item.parentId,
-            dotted_order: task.dottedOrder,
-            trace_id: task.traceId,
-          };
+          if (task.item.run != null) {
+            sampled.push({ dotOrder: task.dotOrder, run: task.item.run });
+
+            if (task.item.interop?.type === "user") {
```

Comment:
What does `type: "user"` represent again?

Reply:
This is set when `runId: string` is passed in `AISDKExporter.getSettings`